arb_os/signedTx.mini

```diff
@@ -87,22 +116,20 @@ impure func recoverSigner(msgData: ByteArray, preSigSize: uint) -> option<addres
         destAddress,
         payment,
         bytearray_extract(msgData, 5*32+1, preSigSize-(5*32+1)),
-        chainParams_chainId(),
-        0,
-        0,
+        v,
+        r,
+        s
     );
-    let rlpHash = keccak256(rlpEncoded, 0, bytearray_size(rlpEncoded));
-
-    let signer = asm(
-        bytearray_get256(msgData, preSigSize),
-        bytearray_get256(msgData, preSigSize+32),
-        1-(bytearray_getByte(msgData, preSigSize+64) % 2),
-        rlpHash,
-    ) address { ecrecover };
+    return keccak256(rlpEncoded, 0, bytearray_size(rlpEncoded));
+}
 
-    if (signer == address(0)) {
-        return None;
-    } else {
-        return Some(signer);
-    }
+public func hashL2messageInfo(l2msg: MarshalledBytes, chainId: uint, sender: address) -> bytes32 {
+    asm(300,) { debugprint };
```

Comment:
I think this debugprint should be removed. Other than that is PR is solid and I have my message batching tested against it and working